src/pages/sign-up/Email/index.tsx

```diff
@@ -71,13 +85,13 @@ export const Email = () => {
             </>
           )}
           <div className="mt-7">
-            <EmailForm />
+            <EmailForm eventId={generateEventsId(eventsItems)[eventName]} />
```

Comment:
What's the point of this function? why you don't just map to ID? since you have used static name for events in this function, I think it causes difficulties in the future